src/components/CustomKeyboard/__tests__/CustomKeyboard.spec.tsx

```diff
@@ -11,14 +11,14 @@ const props = {
   },
   buttonsTextStyle: {
     color: '#CCCCCC',
-    fontSize: 13
+    fontSize: 13,
   },
   cancelKeyText: 'cancelKeyText',
   onCancelPress: jest.fn(),
   onSubmitPress: jest.fn(),
   submitKeyText: 'submitKeyText',
-  visible: false,
-  children: null
+  visible: true,
```

Comment:
why we changed visibility in test?

Reply:
Because in the snapshot there was only root element (the `Modal` component) shown and we wouldn't be able to test anything that was inside that modal. I don't know how they achieved to show inner components with `visible: false` in the original package.

Reply:
React Native made a breaking change in Modal mock where they disabled rendering of children when `visible={false}`: https://github.com/facebook/react-native/pull/32346

Original package was using `0.57.2` where this change wasn't implemented yet.